### ISSUE TITLE
Results on all methods

### ISSUE THREAD

**Issue by lilitao0517** [cantbebetter2/Relational-Diffusion-Distillation-for-Efficient-Image-Generation#1]
Hi author, I have reproduced the source code of rdd provided by you, and found that all the methods in the final test results are slightly lower than the metrics of the paper (FID is more disadvantageous), but it still conforms to the conclusion that RDD is better than the PD and RCFD, even though the results are quite different, is this phenomenon normal?

**Comment by wlfeng0509** [comment]
Hi, did you reproduce the cifar10 or imagenet experiment?

**Comment by lilitao0517** [comment]
> Hi, did you reproduce the cifar10 or imagenet experiment?

I reproduced the experiments about cifar10 and the IS and FID of all methods are off from the paper, but RDD still performs the best.

**Comment by wlfeng0509** [comment]
Did you keep all experiment settings same as the paper setting, like lr and batch_size and other hyperparameters?

**Comment by lilitao0517** [comment]
> Did you keep all experiment settings same as the paper setting, like lr and batch_size and other hyperparameters?

Yes, I checked the corresponding part of the code according to the hyperparameter settings in the paper.

**Comment by wlfeng0509** [comment]
I have uploaded more detailed settings in appendix page in the newest arxiv version, maybe you can align with those settings.
